funnel/extapi/explara.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+
+import requests
+from ..util import format_twitter
+from ..models import SyncTicket
+
+__all__ = ['ExplaraAPI']
+
+
+def strip_or_empty(val):
+    return val.strip() if val else ''
```

Comment:
One of the ways to do this data import cleanly is by passing it through a form so the data mangling parts are nicely isolated. That said, the form is possibly not the best from a performance standpoint, so calling it in a loop like here will slow things down.